Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionVC.swift

```diff
@@ -61,21 +61,21 @@ class EnhancedTrackingProtectionMenuVC: UIViewController {
     }
 
     private let siteDomainLabel: UILabel = .build { label in
-        label.font = ETPMenuUX.Fonts.websiteTitle
-        label.numberOfLines = 0
+//        label.font = ETPMenuUX.Fonts.websiteTitle
+        label.numberOfLines = 2
     }
 
     private var closeButton: UIButton = .build { button in
-        button.backgroundColor = .Photon.LightGrey50
-        button.layer.cornerRadius = 0.5 * ETPMenuUX.UX.closeButtonSize
-        button.clipsToBounds = true
-        button.setImage(UIImage(named: "close-medium"), for: .normal)
-        button.imageView?.contentMode = .scaleAspectFit
+//        button.backgroundColor = .Photon.LightGrey50
```

Comment:
Idea for improvement. I'd think I'd prefer to use block comments /* */ if we comment multiple lines or entire methods. This might help us with future conflict solving. Not a blocker